src/components/Contacts/ContactDetails/ContactDonationsTab/PartnershipInfo/PartnershipInfo.tsx

```diff
@@ -206,7 +206,7 @@ export const PartnershipInfo: React.FC<PartnershipInfoProp> = ({ contact }) => {
           <ClearIcon />
         </IconContainer>
         <LabelsAndText variant="subtitle1" color="textSecondary">
-          {t('Referred by ')}
+          {t('Connecting Partner ')}
```

Comment:
Do you think that this would read better with a colon?

```suggestion
          {t('Connecting Partner: ')}
```

Reply:
Right now the other items don't have colon's. Probably because they are often empty?
<img width="578" alt="Screenshot 2024-09-10 at 12 28 40 PM" src="https://github.com/user-attachments/assets/85236928-4288-47b3-b2e5-ee0cc56a9cb1">

Reply:
Good point. I forgot that the label is a different color, so this is good as-is.